### ISSUE TITLE
Firenvim works in chrome but not firefox

### ISSUE THREAD

**Issue by bangn** [glacambre/firenvim#258]
Thank you very much for creating this plugin. I really enjoy it when it worked.

- OS Version: debian 10.2
- Browser Version: Firefox esr 68.2
- Browser Addon Version: 0.1.14
- Neovim Plugin Version: 0.1.14
- Neovim version: 0.4.2
- What I tried to do:
  * Follow troubleshooting guide
    * Make sure the neovim plugin is installed
    * Make sure the firenvim script has been created
    ```
    echo 'abcde{}' | $XDG_DATA_HOME/firenvim/firenvim
    z{"version": "0.1.14", "settings": {"localSettings": {".*": {"priority": 0, "selector": "textarea,input[type=\"text\"]"}}}}
    ```
  * Make sure the firenvim native manifest has been created -> it is created
    under `$HOME/.mozilla/native-messaging-hosts`
- What happened:
Firenvim works in Chrome but not in Firefox

It was working before however, at some point this week it stops working.

I have tried reinstall neovim plugin as well as the Firenvim plugin. I event
remove `$HOME/.mozilla`. It sill not work.

Follow the debugging step in troubleshooting and navigate to `http://txti.es`
does not show any message.

**Comment by glacambre** [comment]
I think this is happening because I started to use the [ResizeObserver](https://developer.mozilla.org/en-US/docs/Web/API/ResizeObserver) api. I thought it had been implemented in firefox 68 but it looks like it actually requires firefox 69 - I'll implement a workaround. Thanks for reporting this issue.

**Comment by bangn** [comment]
Hi @glacambre 

Thank you very much for your prompt reply

Cheers

**Comment by glacambre** [comment]
Hi, I fixed your issue in #259. I'll publish a new release tomorrow. Thanks for reporting this issue! :)

**Comment by bangn** [comment]
Legend. Thank you very much
